openmrs-19/src/test/java/org/motechproject/openmrs19/it/MRSEncounterServiceIT.java

```diff
@@ -119,6 +119,17 @@ public void shouldCreateEncounter() throws HttpException, ParseException, Interr
     }
 
     @Test
+    public void shouldCreateEncounterWithObservation() throws HttpException, ParseException, InterruptedException {
```

Comment:
you can asserts observation values in test above, there is no need to create another test